hpy/devel/src/runtime/argparse.c

```diff
@@ -53,10 +54,10 @@
  * d (float) [double]
  *     Convert a Python floating point number to a C double.
  *
- * Handles
- * -------
+ * Handles (Python Objects)
+ * ~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * O (object) [HPy *]
+ * O (object) [HPy]
```

Comment:
is it possible to use `fixed width fonts` for things like the character codes and the types? It is especially important for the character codes because with the default font things like `I` risk to be misinterpreted as `l`

Reply:
Good idea & turned out to be quite easy. Fixed.